docs/index.md

```diff
@@ -28,7 +28,7 @@ This tool is intended for use by developers who are experienced with JavaScript.
 ## Including the Buy SDK
 
 ```html
-<script src="http://sdks.shopifycdn.com/js-buy-sdk/latest/shopify-buy.polyfilled.globals.min.js"></script>
+<script src="http://sdks.shopifycdn.com/js-buy-sdk/v0/latest/shopify-buy.umd.polyfilled.min.js"></script>
```

Comment:
I agree there's a problem (people linking to `latest`) but i seriously doubt this is a fix. People linking to `latest` do that because they ignorantly copy and paste the link we provide or they are just lazy to find out what the most recent link are. If the *most recent* version is what's presented on the doc, they'd do the same with it.

Prepending a `major` version to `latest` will also potentially cause confusion. Or is this a normal thing with open source projects?

Except `v0` is being added to explicitly support 'bleeding per major version' and not to try to fix the problem, i don't think it's necessary.

As per fixing the problem, no way comes to mind right now ... other than ensuring the proper version is presented on the documentation.